code/modules/cargo/packs.dm

```diff
@@ -982,8 +982,10 @@
 					/obj/item/reagent_containers/pill/insulin,
 					/obj/item/stack/medical/gauze,
 					/obj/item/storage/box/beakers,
+					/obj/item/storage/box/medsprays,
 					/obj/item/storage/box/syringes,
-				    /obj/item/storage/box/bodybags)
+					/obj/item/storage/box/medsprays,
```

Comment:
Keep identical entries next to each other, you've got a box of syringes between boxes of medsprays.

Reply:
Whoops, good catch. The double entry was a mistake that happened during merge conflict resolution. Fixed now.